### PR TITLE
get ready for sec

### PR DESCRIPTION
**Pull request by martacki**
@SermishaNarayana

Creating here a PR that allows to execute the sector coupeled version of PyPSA-Eur as power only model for UA. This will extend the model to:
- display the distribution grid
- display solar rooftop PV
- move all power demands from the transmission level to distribution level
- some other artifacts that are nice to have, but not our focus

**But**:
Needs a lot of testing, so that all the features you implemented for the elec only model, mainly:
- reserve margins
- addition of existing plants and their maintenance schedules

### CODE REVIEW

File: Config/config.open_model.yaml [hunk 1]
```diff
@@ -85,8 +87,8 @@ enable:
   retrieve_sector_databundle: false
   retrieve_cost_data: true
   retrieve_cutout: false
-  build_cutout: false
-  custom_busmap: true
+  build_cutout: true
+  custom_busmap: false
```
Comment (martacki) [inline Config/config.open_model.yaml:91]: needs to be modified back, but I set it to false b/c the sector coupeled model does not allow the busmap names to differ from the standard naming scheme of

2 letters for country + 1 number for synch. zone + space + number for cluster, e.g.
`UA0 1`

File: Config/config.open_model.yaml [hunk 2]
```diff
@@ -102,6 +104,8 @@ solving:
 clustering:
   simplify_network:
     remove_stubs: false
+  temporal:
+    resolution_sector: 500H
```
Comment (martacki) [inline Config/config.open_model.yaml:108]:
```suggestion
    resolution_sector: 24H
```

**Comment by yerbol-akhmetov** [comment]
@martacki, tested, works with https://github.com/open-energy-transition/pypsa-eur/pull/1. 

However, all functions in `00_runscenarios.py` needs to be modified to remove hard-coded filename (e.g. `add_self_consumption`, `add_regional_demands`), @SermishaNarayana.

**Comment by yerbol-akhmetov** [comment]
I made changes to `00_runsimulation.py` locally and managed to run sector version. It runs without error. However, the electricity mix is different than expected (does not have coal). Probably I missed something. It needs review. I can push here the changes. Ping me @SermishaNarayana if you want to start from scratch or use my changes.

**Comment by martacki** [comment]
> I made changes to `00_runsimulation.py` locally and managed to run sector version. It runs without error. However, the electricity mix is different than expected (does not have coal). Probably I missed something. It needs review. I can push here the changes. Ping me @SermishaNarayana if you want to start from scratch or use my changes.

Yes, capacities in the sector model are lost, as we don't keep them from the existing model. You can check the capacities probably, they should be 0 atm. merging, then.

**Pull request merged**